docs/monitoring.md

```diff
@@ -222,17 +222,40 @@ both running applications, and in the history server.  The endpoints are mounted
 for the history server, they would typically be accessible at `http://<server-url>:18080/api/v1`, and
 for a running application, at `http://localhost:4040/api/v1`.
 
+In the API, an application is referenced by its application ID, `[app-id]`.
+When running on YARN, each application may have multiple attempts; each identified by their `[attempt-id]`.
+In the API listed below, `[app-id]` will actually be `[base-app-id]/[attempt-id]`,
+where `[base-app-id]` is the YARN application ID.
+
 <table class="table">
   <tr><th>Endpoint</th><th>Meaning</th></tr>
   <tr>
     <td><code>/applications</code></td>
     <td>A list of all applications</td>
   </tr>
   <tr>
+    <td></td>
+    <td><code>?status=[completed|running]</code> list only applications in the chosen state</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td><code>?minDate=[date]</code> earliest date/time to list. Examples:
```

Comment:
Formatting-wise, nit, can this be accomplished with `<p>` or `<br/>` in one big `<td>` rather than make extraneous rows, or does it just not look nice?